src/components/Compare/Compare.style.tsx

```diff
@@ -223,13 +214,36 @@ export const ArrowContainer = styled.div<{
   }
 `;
 
-export const Footer = styled.div`
+export const Footer = styled.div<{ isCounty: any }>`
```

Comment:
do you want to make `isCounty: boolean`?

Reply:
its not a boolean-- isCounty={props.county}, which is either a location object or null